### PR TITLE
Renaming placeholder labels in maps folder

### PR DESCRIPTION
**Pull request by DeeDeeG**
Open to feedback while I work on these.

I tend to rename NPCs to more specific names or fix incorrect NPC names where it makes sense to. Let me know if this is done right or you have any feedback on any of this.

Thanks.

### CODE REVIEW

File: maps/PowerPlant.asm [hunk 1]
```diff
@@ -404,7 +404,7 @@ PowerPlant_MapEvents:
 	object_event  4, 14, SPRITE_OFFICER, SPRITEMOVEDATA_STANDING_DOWN, 0, 0, -1, -1, PAL_NPC_GREEN, OBJECTTYPE_SCRIPT, 0, PowerPlantOfficerScript, -1
 	object_event  2,  9, SPRITE_GYM_GUY, SPRITEMOVEDATA_SPINRANDOM_SLOW, 0, 0, -1, -1, PAL_NPC_BLUE, OBJECTTYPE_SCRIPT, 0, PowerPlantGymGuy1Script, -1
 	object_event  6, 11, SPRITE_GYM_GUY, SPRITEMOVEDATA_STANDING_UP, 0, 0, -1, -1, PAL_NPC_BLUE, OBJECTTYPE_SCRIPT, 0, PowerPlantGymGuy2Script, -1
-	object_event  9,  3, SPRITE_OFFICER, SPRITEMOVEDATA_STANDING_LEFT, 0, 0, -1, -1, PAL_NPC_GREEN, OBJECTTYPE_SCRIPT, 0, PowerPlantGymGuy3Script, -1
+	object_event  9,  3, SPRITE_OFFICER, SPRITEMOVEDATA_STANDING_LEFT, 0, 0, -1, -1, PAL_NPC_GREEN, OBJECTTYPE_SCRIPT, 0, PowerPlantOficer2Script, -1
```
Comment (Rangi42) [inline maps/PowerPlant.asm]: Good catch, but: "Oficer" → "Officer".

**Comment by DeeDeeG** [comment]
_(More `maps/*.asm` files to be updated as/when I get time...)_

**Comment by DeeDeeG** [comment]
Something kind of odd in GoldenrodUnderground.asm:

The OlderHairCutBrotherScript does `special YoungerHaircutBrother`

https://github.com/pret/pokecrystal/blob/0ef91b912905e3a297b75abc2694f99881221ed0/maps/GoldenrodUnderground.asm#L185-L204

And the YoungerHaircutBrotherScript does `special OlderHaircutBrother`

https://github.com/pret/pokecrystal/blob/0ef91b912905e3a297b75abc2694f99881221ed0/maps/GoldenrodUnderground.asm#L268-L287

I suspect the names of these functions/events are swapped in our codebase.

**Comment by DeeDeeG** [comment]
I can't figure out how to fix the haircut event names, so I'll leave that for somebody else, probably in a separate PR.

**Comment by Rangi42** [comment]
@DeeDeeG Can this be merged now, and the haircut event names fixed separately? Or do you plan to add more maps to this PR?

**Comment by DeeDeeG** [comment]
Hi @Rangi42.

I am in the habit of posting the PR early, and updating as I go.

As it turns out, I have been busy, so I can't add any more maps soon. (Also: I don't have enough understanding of the programming logic to properly fix the haircut names myself. I tried changing it a number of ways, but all failed the Travis CI check.)

But if you would like to merge this, now, that's fine.

**Comment by DeeDeeG** [comment]
Renamed the Haircut Brothers NPC constant names... I checked the schedule they appear against Bulbapedia...
https://bulbapedia.bulbagarden.net/wiki/Haircut_brothers#In_the_games

**Comment by DeeDeeG** [comment]
I don't really get what's going on here: 
https://github.com/pret/pokecrystal/blob/2d5d364c061e5046d2ead78c8f6544e137842744/maps/GoldenrodCity.asm#L548-L550

This UnknownText is used in each of the `.Flamethrower`, `.Thunderbolt`, and `.IceBeam` subroutines of the GoldenrodCity.asm Move Tutor script... for example:
https://github.com/pret/pokecrystal/blob/2d5d364c061e5046d2ead78c8f6544e137842744/maps/GoldenrodCity.asm#L74-L79

**Comment by Rangi42** [comment]
> I don't really get what's going on here:

Try removing that particular `writetext` and then playing the move tutor event, and you'll see what it's for. It clears the textbox before the move selection screen appears. Anyway, if you need a proper label for it, `...BlankText` or `...ClearTextbox` would be fine. (Note the recent standardization of capital letters: `Textbox`, not `TextBox`.)

**Comment by DeeDeeG** [comment]
How's this: `GoldenrodCityMoveTutorClearTextbox` As in...

```assembly
.Flamethrower:
	setval MOVETUTOR_FLAMETHROWER
	writetext GoldenrodCityMoveTutorClearTextbox
	[ . . . ]
```

**Comment by DeeDeeG** [comment]
Hi folks. I think this is at a good point to review and/or merge. Don't want to get too deep into it before getting some feedback or noticing issues that would need to be addressed, etc.

Hopefully I will have some time to do more of these.

**Pull request merged**